server/frontend/src/components/Bugs/Comments/PublicationForm.vue

```diff
@@ -257,6 +258,7 @@ export default {
     crashData: "",
   }),
   async mounted() {
+    Base64.extendString();
```

Comment:
I'm worried about extending the String prototype as this might cause issues later that aren't inherently clear.  Is there a reason you did this as opposed to `Base64.encode(str)`?

Reply:
No particular reason, you're right, I'll change for `Base64.encode(str)`.